.changes/0821-chat-fixes.md

```diff
@@ -0,0 +1,4 @@
+- [fix] activate chat reactions.
+- [improvement] chat reactions ui.
+- [fix] reply content fixes.
+- [fix] chat list duplication bug.
```

Comment:
Let's try to give more insights to actual end-users with our changes recordings (this isn't just for developers). How about:

```suggestion
- **Chat message emoji-reactions**: we now render chat-message reactions again, as well as allow you to send them by clicking in the UI.
- Chat replies fixed: rendering replies as well as composing them has been redesigned and is a lot more clear now
- messages appearing twice in a chat room should be fixed now. If you see that again, please let us know.
```